docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -882,63 +882,175 @@
 
-        <para>Where the compiled ECL queries are stored. The storage needs to
-        allow shared objects to be directly loaded from it efficiently.</para>
+          <listitem>
+            <para><emphasis role="bold">sasha:</emphasis> Stores archived
+            workunits and similar data. Performance requirements are typically
+            lower, allowing use of lower-cost storage.</para>
+          </listitem>
 
-        <para>If you wanted both Dali and dll data on the same plane, it is
-        possible to use the same prefix for both subpath properties. Both
-        would use the same prefix, but should have different subpaths.</para>
+          <listitem>
+            <para><emphasis role="bold">spill:</emphasis> (<emphasis
+            role="italic">Optional</emphasis>) Stores spill files. Local NVMe
+            disks are potentially a good choice for this.</para>
+          </listitem>
+
+          <listitem>
+            <para><emphasis role="bold">temp:</emphasis> (<emphasis
+            role="italic">Optional</emphasis>) Stores internal temporary spill
+            files created by the engines. If it is undefined, default to the
+            "spill" category plane. Like the spill plane this should generally
+            be backed by fast storage (like local NVMe). It is also worth
+            noting that the "spill" and "temp" planes are never read remotely,
+            e.g. from other nodes. Unlike the others which need to be
+            accessible globally. This is why local NVMe storage is
+            suitable.</para>
+          </listitem>
+        </itemizedlist>
       </sect3>
 
-      <sect3>
-        <title>sasha</title>
+      <sect3 id="multiple-device-planes">
+        <title>Multiple Device Planes</title>
 
-        <para>This is the location where archived workunits, etc are stored
-        and it is typically less speed critical, requiring lower storage
-        costs.</para>
-      </sect3>
+        <para>HPCC Systems storage planes are typically backed by a single
+        Persistent Volume Claim (PVC). If you wanted to increase the overall
+        bandwidth, you can do so using multiple "devices". You can configure a
+        storage plane to use multiple PVCs. It's also possible to achieve same
+        with <emphasis>storageapi.containers</emphasis> (see
+        values.schema.json).</para>
 
-      <sect3>
-        <title>spill</title>
+        <para>To configure an HPCC Storage Plane to be backed by multiple
+        PVCs:</para>
+
+        <itemizedlist>
+          <listitem>
+            <para>Set the <emphasis>numDevices</emphasis> property to the
+            number of PVCs you want to use. </para>
+
+            <programlisting language="yaml">- name: data
+  pvc: data-mystorage-hpcc-localfile-pvc
+  prefix: "/var/lib/HPCCSystems/hpcc-data"
+  category: data
+  numDevices: 2        </programlisting>
+
+            <para>The Helm chart will expect to find PVCs named using the base
```

Comment:
We should also tell at reach index the append is expected to start (i.e., will it start at 0 or 1?)